VeloC plugin (#661)
pdidev/pdi: CHANGELOG.md

* Update the error messages for invalid SYSTEM dependencies of NetCDF and HDF5,
  and document the cmake option to override 'NetCDF_FIND_STRATEGIES' as
  requested in [#500](https://github.com/pdidev/pdi/issues/500)

> * Added new VeloC plugin. Enables users to write and recover checkpoints using the [VeloC library](https://veloc.readthedocs.io/en/latest/userguide.html)

Reviewer: ```suggestion
* Added new checkpointing plugin using the [VeloC library](https://veloc.readthedocs.io/en/latest/userguide.html) as requested in "..."
```
Please create an issue and link here

* Update the error messages for invalid SYSTEM dependencies of NetCDF and HDF5,
  and document the cmake option to override 'NetCDF_FIND_STRATEGIES' as
  requested in [#500](https://github.com/pdidev/pdi/issues/500)

> * Added new checkpointing plugin using the [VeloC library](https://veloc.readthedocs.io/en/latest/userguide.html) as requested in [issue #726](https://github.com/pdidev/pdi/issues/726)

Reviewer 2: ```suggestion
* Added new checkpointing plugin using the [VeloC library](https://veloc.readthedocs.io/en/latest/userguide.html)
as requested in [#726](https://github.com/pdidev/pdi/issues/726)
```

Reviewer 2: To reduce the number of character by line as it is done for other entry.
